bgpd/bgp_clist.c

```diff
@@ -823,6 +823,7 @@ struct community *community_list_match_delete(struct community *com,
 	/* Delete all of the communities we flagged for deletion */
 	for (i = delete_index - 1; i >= 0; i--) {
 		val = community_val_get(com, com_index_to_delete[i]);
+		val = htonl(val);
```

Comment:
Confused as to why this is necessary - brief summary would be helpful

Reply:
@qlyoung added small note in commit message.